src/config/n_plus_one.rs

```diff
@@ -20,7 +20,7 @@ pub fn n_plus_one(config: &Config) -> Vec<Vec<(String, String)>> {
           } else if field.has_resolver() && !field.has_batched_resolver() && is_list {
             vec![new_path]
           } else {
-            find_fan_out(config, &field.type_of, new_path, field.list.is_some() || is_list)
+            find_fan_out(config, &field.type_of, new_path, field.list.unwrap_or(false) || is_list)
```

Comment:
```suggestion
            find_fan_out(config, &field.type_of, new_path, field.isList() || is_list)
```
Would really appreciate if we could add a `isList` operator on config. This will keep the code clean.

Reply:
The branch is already merged. Do you want to see this as a separate PR, or would you prefer it to be covered by another approach from the https://github.com/tailcallhq/tailcall/issues/367?